### DISCUSSION TITLE
What is the question mark next to a branch name?

### DISCUSSION THREAD

**Discussion by sayandcode** [jesseduffield/lazygit#2353] [Q&A]
I'm currently using worktrees to have two branches running in two separate files. In one of the branches however, I see that there is a wierd question mark next to the branch name. Here's some more context.
![Screenshot from 2023-01-09 15-26-45](https://user-images.githubusercontent.com/102280939/211282482-0aaf8964-e70f-41a5-8932-0e8b4b8f0480.png)
![Screenshot from 2023-01-09 15-24-34](https://user-images.githubusercontent.com/102280939/211282712-717f6835-7586-4b05-81b4-131637d0c538.png)
![Screenshot from 2023-01-09 15-26-55](https://user-images.githubusercontent.com/102280939/211282489-845f3fa3-4043-42c8-8ed1-2756e19fd349.png)
However, when I check the **main branch that is frontend** this question mark is not present in that branch
![Screenshot from 2023-01-09 15-27-08](https://user-images.githubusercontent.com/102280939/211282492-212bc375-d591-4ce6-a1f8-411154230821.png).
I have tried syncing remote and local, amongst other things, but the question mark doesn't go away.

What does that question mark signify?

**Comment by on3iro** [comment]
same issue here (also related to worktrees)

**Reply by Ryooooooga** [reply]
the remote branch is not stored locally.

**Reply by on3iro** [reply]
ah, is there a way to fix this with work trees @Ryooooooga ? Things seem to work mostly fine, however I noticed that I can't force push from inside lazygit from work trees (the regulal git cli does work without issues).

**Reply by on3iro** [reply]
i've found a good solution, someone wrote a nice script to make cloning bare repos compatible with worktrees easier:
https://morgan.cugerone.com/blog/workarounds-to-git-worktree-using-bare-repository-and-cannot-fetch-remote-branches/
